modules/3d/perf/perf_rendering.cpp

```diff
@@ -0,0 +1,279 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+#include "perf_precomp.hpp"
+
+namespace opencv_test
+{
+
+// that was easier than using CV_ENUM() macro
```

Comment:
really?

Reply:
When talking about `enum class` this is 100% true.
For ordinary enum our macro is more convenient but even though:
* it doesn't have an operator to convert to the original enum type, just to int, which leads to compile errors
* it generates bulky names with prefixes and so on
* it does not protect you from listing several values with the same corresponding int value

I think we need a better enum macro. But until we have one, I'd better use my workaround.